### ISSUE TITLE
Accessibility: Teaching Page

### ISSUE THREAD

**Issue by JonEsparaz** [themeetinghouse/web#353]
Most elements on the teaching page (e.g. scroll list videos, the new banner cards) aren't keyboard focusable. Wrapping in `<a>` tags should do the trick.

**Comment by paul-ooi** [comment]
Hi @JonEsparaz , I spoke briefly with @deeinstereo a while ago about pitching in regarding some accessibility items. 

Would like to help with this. I'm trying to familiarize myself with the code too. Dunno if necessary to assign this to me or just wait for my pull request.

Although `<a>` would make these items focus-able, semantically since these areas are more interactive in nature, `<button>` would be more appropriate (along with the click events too on these items).

**Comment by JonEsparaz** [comment]
@paul-ooi agreed buttons are more appropriate for this use case. The banner cards though, act more like links and should make use of the [Link component](https://github.com/themeetinghouse/web/blob/master/src/components/Link/Link.tsx) that was recently implemented.

I've assigned you this issue. Let me know if you have any questions about the code. Looking forward to the PR.

**Comment by paul-ooi** [comment]
Some questions about the code, but aren't directly related to this Issue, not sure where to add or open a new issue(s).

I'm using Voiceover (mac), NVDA(windows) screen readers to check the experience and results of my branch.

1. I've modified my branch with the Link Component for the banner cards 
1. I've also made the buttons, but should this also be using the Link Component's `<LinkButton>` ?
1. While testing the buttons are just using the visible text (see screenshots), so I would suggest additional text for a more informative/accessible experience "Play [insert visible text] video in Modal"
![Teaching banner](https://user-images.githubusercontent.com/27851387/92416890-955ddb80-f12d-11ea-8909-02a3cace0325.jpg)
![latest teaching](https://user-images.githubusercontent.com/27851387/92416889-955ddb80-f12d-11ea-9e46-5160d8412e7a.jpg)
1. the video number seems confusing as a visitor (maybe I'm missing the meaning behind this number, I saw something in the code about sorting, ex. '38. Real Security')
1. a lot of images are reading alt text as TBD
1. trying to clone the repo to my Windows computer it encountered an error, I think it may be because the image filenames, some have the colon character, thankfully I'm troubleshooting on my Linux and using the dev server to test on my windows pc
1. I noticed a lot of `outline: none` on elements, so I dunno if that's apart of the design system but it is an accessible guideline ([2.4.7 Focus Visible - Level AA](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=247#focus-visible))

**Comment by JonEsparaz** [comment]
@paul-ooi

> 1. I've modified my branch with the Link Component for the banner cards
- Great 👍
> 2. I've also made the buttons, but should this also be using the Link Component's `<LinkButton>` ?
- Normal buttons are fine for the scroll lists. You'll see that the handleClick function changes the overlayData state variable, then pushes to the history stack, so it isn't really a link.
> 3. While testing the buttons are just using the visible text (see screenshots), so I would suggest additional text for a more informative/accessible experience "Play [insert visible text] video in Modal"
- If you want to edit the visible text, then we'll need @deeinstereo's input.
> 4. the video number seems confusing as a visitor (maybe I'm missing the meaning behind this number, I saw something in the code about sorting, ex. '38. Real Security')
- Those are episode numbers. The majority of users on the teaching page regularly watch our teaching videos and will understand the meaning.
> 5. a lot of images are reading alt text as TBD
- New issue: https://github.com/themeetinghouse/web/issues/439
> 6. trying to clone the repo to my Windows computer it encountered an error, I think it may be because the image filenames, some have the colon character, thankfully I'm troubleshooting on my Linux and using the dev server to test on my windows pc
- Microsoft recommends WSL for NodeJS development: https://docs.microsoft.com/en-us/windows/nodejs/setup-on-wsl2.
> 7. I noticed a lot of `outline: none` on elements, so I dunno if that's apart of the design system but it is an accessible guideline ([2.4.7 Focus Visible - Level AA](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=247#focus-visible))
- Here's our design system: https://www.figma.com/file/BXPAUnmoz9SDF17nhUUWwJ/TMH-UI---GitHub-Sharing?node-id=1%3A2. There's obviously some flexibility if it means accessibility improvements, so feel free to make changes as you see fit. We'll check to make sure everything is ok in terms of styling when you submit the PR.

**Comment by deeinstereo** [comment]
@paul-ooi thanks for jumping in on this! Looks like you and @JonEsparaz got a handle on things! Let me know if there's anything you need from me.
